docs/orchestration/execution/storage_options.md

```diff
@@ -143,6 +143,28 @@ GitLab storage uses a [personal access token](https://docs.gitlab.com/ee/user/pr
 GitLab server users can point the `host` argument to their personal GitLab instance.
 :::
 
+## Bitbucket
```

Comment:
Could you move this to /orchestration/flow_config/storage.html? That is a new page where these details are organized

Reply:
I went ahead and copied it into flow_config/storage.md. Should I delete it from the exectution/storage_options.md?